src/app/api/fetchNews.ts

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+import { API_KEY, API_URL, ResponseAPI } from "./constants";
+
+export async function fetchNews(): Promise<ResponseAPI> {
+  try {
+    const url = `${API_URL}top-headlines?language=en&pageSize=18&apiKey=${API_KEY}`;
+    const { data } = await axios<ResponseAPI>(url);
+    return data;
+  } catch (err) {
```

Comment:
What is the point in catching an error, do nothing and then throw the same error? If you handle this error in a different place, please remove try catch

Reply:
But initially i had handled this error in different place, but you said that it's need to be try catch there